synapse/events/__init__.py

```diff
@@ -159,6 +159,12 @@ def items(self):
     def keys(self):
         return six.iterkeys(self._event_dict)
 
+    def prev_event_ids(self):
```

Comment:
Are these ordered? Should we always return them as a set, vs a list?

also docstrings plz

Reply:
So many good points to counter my crankiness

Reply:
(I also ask wrt set because a set comprehension will mean we don't have to cast it to a set in various places, if we declare the events have no fixed order)